index.html

```diff
@@ -343,8 +343,11 @@
           <li>If the <a>browsing context</a> of the script calling the
           constructor is a <a>nested browsing context</a> whose origin is
```

Comment:
> I think this PR should be blocked on fixing these issues:
> #324
> #332

Agreed. Though I wasn’t sure that this PR itself was the right place to fix those. I guess I imagined this PR branch could be rebased once those fixes got made. But I suppose we could instead make the changes on this branch as part of this PR.